### DISCUSSION TITLE
How to change the text "Select an Options" in select component

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#9463] [Help] [answered]
Hi All,
How to change the text "Select an Options" in select component,
Let say just become "Select Block"

![image](https://github.com/filamentphp/filament/assets/6214964/6b6d114b-75d5-4c0a-b217-ba5b07f19605)

Thank you

**Comment by leandrocfe** [comment, marked as answer]
```php
->placeholder('Select Block')
```
